arangod/Agency/Node.cpp

```diff
@@ -502,22 +502,18 @@ ResultT<std::shared_ptr<Node>> Node::handle<SET>(VPackSlice const& slice) {
 /// Increment integer value or set 1
 template <>
 ResultT<std::shared_ptr<Node>> Node::handle<INCREMENT>(VPackSlice const& slice) {
-  size_t inc = (slice.hasKey("step") && slice.get("step").isUInt())
-                   ? slice.get("step").getUInt() : 1;
+  int inc = (slice.hasKey("step") && slice.get("step").isUInt())
+                   ? slice.get("step").getInt() : 1;
```

Comment:
Why do you check `isUInt()`, but then call `getInt()`? Plus `int inc` should probably be `int64_t inc`, or just `auto inc`.